codingTest_김준우/문자열 내림차순으로 배치하기.js

```diff
@@ -0,0 +1,4 @@
+function solution(s) {
+    answer = s.split("").sort().reverse().join("");
```

Comment:
```suggestion
    const answer = s.split("").sort().reverse().join("");
```
const나 let을 붙여주지 않으면 암묵적으로 var로 선언되기 때문에 꼭 붙여주세요. 위 같은 상황에선 const가 맞겠습니다.